stubs/python-jose/jose/jws.pyi

```diff
@@ -4,21 +4,21 @@ from typing import Any
 from .backends.base import Key
 
 def sign(
-    payload: str | Mapping[str, Any],
+    payload: bytes | Mapping[str, Any],
     # Internally it's passed down to jwk.construct(), which explicitly checks for
     # key as dict instance, instead of a Mapping
-    key: str | dict[str, Any] | Key,
+    key: str | bytes | dict[str, Any] | Key,
     headers: Mapping[str, Any] | None = ...,
     algorithm: str = ...,
 ) -> str: ...
 def verify(
-    token: str,
-    key: str | Mapping[str, Any] | Key,
+    token: str | bytes,
```

Comment:
It is passed down to `_load`, which accounts for both `str` and `bytes`.
- https://github.com/mpdavis/python-jose/blob/96474ecfb6ad3ce16f41b0814ab5126d58725e2a/jose/jws.py#L70
- https://github.com/mpdavis/python-jose/blob/96474ecfb6ad3ce16f41b0814ab5126d58725e2a/jose/jws.py#L173-L174